animareflection/ui
# Modal Component

## Description

##### Task link: https://trello.com/c/k9LYz4Z7/116-modal-component

Added custom `Modal` component with `trigger`, `title`, and `description` props. Also added primitive exports to allow for other users to build and customize `Modal` components outside of AR branding/usage.

## Test Steps

1) Verify that everything renders appropriately in storybook
2) Verify that all tests pass
3) Verify that `Modal` usage in downstream app works as expected
4) Verify that `primitives` module gets bundled appropriately, and that users may construct custom `Modal` components in a downstream app.
